### DISCUSSION TITLE
Framework-agnostic Lumberjack release plan

### DISCUSSION THREAD

**Discussion by LayZeeDK** [ngworker/lumberjack#167] [General]
At an open meeting in [the public ngworkers Discord server](https://discord.gg/wwjhWyx7p8) on 2023-08-05, the team drafted the following release plan for transitioning from the Angular-specific [`@ngworker/lumberjack`](https://npmjs.com/package/@ngworker/lumberjack) package to `@lumberjackjs`-scoped npm packages.

1. New packages pull request (#154)
   a. Implement new packages
   b. Write guide to migrating from `@ngworker/lumberjack`
   c. Merge pull request
2. Transfer Lumberjack repo from the [ngworker](https://github.com/ngworker) GitHub organization to the [lumberjack-js](https://github.com/lumberjack-js) GitHub organization
3. Update tooling
   a. Discord bots   
   b. SonarCloud
   c. Docusaurus
4. Consider [RFCs](https://github.com/ngworker/lumberjack/discussions/categories/rfc)
5. Release new packages
   a. Update metadata
      1. Git repository
      2. Issues
      3. Website
      4. Badges

   b. Publish to npm
      1. Core packages: CommonJS and ESM
      2. Angular packages: APM

   c. Tag GitHub releases
6. Deprecate the [`@ngworker/lumberjack`](https://npmjs.com/package/@ngworker/lumberjack) package on npm, referring to the new Angular-specific package, `@lumberjackjs/angular`.

**Comment by NachoVazquez** [comment]
Sounds like a plan!

**Comment by NachoVazquez** [comment]
After the transference to the `lumberjack-js` organization, we also need to:

- Update the Sonar Cloud on the platform
    - Update the configuration of the project
- Update the Docusaurus release path
- Update code references to the old repository (for example, in the README badges)

**Comment by NachoVazquez** [comment]
Closing this discussion as we decided to move in a different direction, see #188
